src/snippets/product-card--empty.liquid

```diff
@@ -8,11 +8,16 @@
       {{ 'product-' | append: index | placeholder_svg_tag: 'placeholder-svg' }}
     </div>
     <div class="product-item__box">
+      {% if display_vendor %}
+        <h4 class="product-item__subtitle">
```

Comment:
@rdnvol now you do not have this variable, but it is better to leave this name and change where you pass it.

Reply:
@IhorTretiy 
I'm not sure I understood you correctly.
Do you mean that I have to rename the variable in the render tag?

Reply:
@rdnvol yes, that's what I meant